megamek/src/megamek/server/Server.java

```diff
@@ -1095,6 +1080,7 @@ private IPlayer addNewPlayer(int connId, String name) {
     public void validatePlayerInfo(int playerId) {
         final IPlayer player = getPlayer(playerId);
 
+        // TODO : Windchild can this be removed?
         // maybe this isn't actually useful
         // // replace characters we don't like with "X"
         // StringBuffer nameBuff = new StringBuffer(player.getName());
```

Comment:
Can we remove this area of commented out code?

Reply:
This was commented out over ten years ago. I'm going to go out on a limb and say it's safe to get rid of it.